### PR TITLE
Home page 

### PR DESCRIPTION
**Pull request by rahul0396**
https://user-images.githubusercontent.com/43301962/154839155-9659f1c8-521f-419d-9683-c0e97f767242.mp4

### CODE REVIEW

File: css/homepage.css [hunk 1]
```diff
@@ -0,0 +1,84 @@
+@import url(/css/navbar.css);
+
+
+
```
Comment (saadahmadhere) [inline css/homepage.css:4]: extra spacing should be removed

File: css/homepage.css [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+
+/* slider css */
+.slider-box {
+    width : 80%;
+    height: 100%;
+    max-height: 100vh;
+    position  : relative;
+
```
Comment (saadahmadhere) [inline css/homepage.css:12]: here too

File: css/homepage.css [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+    transition: 0.7s;
+}
+
+#slider img {
+    /* overflow: hidden; */
+    width: 100%;
+}
+
+.indicators {
+    /* position : absolute; */
```
Comment (saadahmadhere) [inline css/homepage.css:29]: unwanted comments should be removed

File: css/homepage.css [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+    height       : 15px;
+    border-radius: 50%;
+    background   : rgba(255, 255, 255, 0.5);
+    cursor       : pointer;
+}
+
+.active {
+    background: #ffff !important;
+}
+/* slider css end */
```
Comment (saadahmadhere) [inline css/homepage.css:47]: no need to specify end of a certain class styling. The beginning of the new class marks the end of the previous one.

File: css/homepage.css [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+.card{
+    width: 25rem;
+    height: 80%;
+    /* margin: 1rem; */
+}
+.card-head{
+    margin: 3px;
+}
+
+/* .category-card-container{ */
```
Comment (saadahmadhere) [inline css/homepage.css:77]: unwanted comments

File: css/navbar.css [hunk 6]
```diff
@@ -0,0 +1,56 @@
+
+
```
Comment (saadahmadhere) [inline css/navbar.css:2]: unwanted space.

File: pages/homepage.html [hunk 7]
```diff
@@ -0,0 +1,253 @@
...
+                </span>
+    
+                <div class="card-text-overlay-row">
+                    <!-- <img class="card-horizontal-img" src="..." alt="..." /> -->
+                    
+                    <div>
+                        <h1 class="card-head">
+                            Mass Gainer
+                        </h1>
+                        <!-- <p class="card-text">Peanut Butter</p> -->
```
Comment (saadahmadhere) [inline pages/homepage.html:162]: These kinds of unwanted comments should be removed.

**Review by saadahmadhere — CHANGES_REQUESTED** [review]
Well written, very clean code. However, there are some unwanted space and comments which should be removed.

**Review by Ronak-Mutha — COMMENTED** [review]
Remove extra comments and you can use variables for colors and spacing. Rest all good to merge.

Note: These are just suggestions, try to work on them after submission [Not required, good to have.]

**Pull request merged**